index.html

```diff
@@ -2187,6 +2190,72 @@ <h2>Denial of Service</h2>
                     </dd>
                 </dl>
 	</section>
+        <section id="security-consideration-amp-ddos">
+            <h2>Amplification and Distributed Denial of Service</h2>
+            <p>
+	    It may also be possible to use elements of WoT Discovery
+	    mechanisms to launch distributed denial of service attacks
+	    against other targets.  In such an attack the WoT Discovery
+	    service itself is not the target.  Instead, an aspect of
+	    the WoT Discovery service would be
+	    exploited to generate network traffic that overloads a third
+	    party, the actual target.
+	    Such an attack has two requirements: first, the ability
+	    to redirect traffic to a third party, and second, 
+	    an intermediary service
+	    that can be exploited to amplify network traffic
+	    from the attacker.
+	    Redirection
+	    of network traffic is possible in some protocols, such as 
+	    unsecured CoAP, by modifying source information in headers.
+	    Amplification is possible by taking advantage of three 
+	    multiplicative factors: the ratio of request to response payload
+	    sizes, use of observe in protocols like CoAP  
+	    (which can give multiple results for one request), and
+	    multicast (which can allow multiple servers to respond to one 
+	    request). 
+	    Services which do not support authentication
+	    are ideal intermediaries for such an indirect attack.
+	    Unfortunately, the Introduction mechanisms for WoT Discovery
+	    are meant to provide open access mechanisms to initiate discovery
+	    and might be exploited for this purpose.
+            </p>
+            <dl>
+                <dt>Mitigations:</dt>
+                <dd>
+		To mitigate this problem, the following is recommended:
+		<ul>
+		<li>Disable support for observe or similar extended result subprotocols
+		on Introduction mechanisms.</li>
+		<li>Introduction mechanisms should not respond to multicast
+		requests unless this is absolutely required by the protocol.</li>
```

Comment:
In the case of CoAP, there is a new RFC ([RFC 9175](https://datatracker.ietf.org/doc/html/rfc9175)) that seems to mitigate the amplification potential of multicast requests and other use cases.

Reply:
Using the new RFC, the bullet point might be adjusted like so: 

```suggestion
		<li>In general, introduction mechanisms should not respond to multicast
		requests unless this is absolutely required by the protocol.
		In the case of CoAP, the recommendations made in [[RFC9175]] should be
		applied in order to mitigate risks of amplification attacks.
		</li>
```

I need to read a little bit more thoroughly thorugh the RFC myself, though.

Reply:
Hmm, nevermind, I just read through [draft-mattsson-t2trg-amplification-attacks](https://datatracker.ietf.org/doc/draft-mattsson-t2trg-amplification-attacks/) where the same authors state that

> [f]or multicast requests, anti-amplification limits and the Echo Option do not really work unless the number of servers sending responses is known.

Maybe RFC 9175 could be cited elsewhere, though?

Reply:
yeah, they mentioned that too, you generally don't know the number of servers.  So, discouraging multicast is probably best, but it does mean that doing a multicast as part of an introduction (which is actually useful...) is a problem, but in an OPEN network (as opposed to a closed, trusted network).  Anyway, let me think about it, should probably cite that RFC and mention the issue that we don't know the number of servers, generally.

Reply:
Can still SAY what you propose, but temper a bit to say "If multicast is necessary..."

Reply:
ok, updated, pls take a look

Reply:
Thank you, @mmccool, looks good! :)